wdl/tasks/CallVariantsPBCCS.wdl

```diff
@@ -134,6 +147,20 @@ workflow CallVariants {
             prefix   = prefix + ".deepvariant_pepper"
     }
 
+    call VariantUtils.MergePerChrCalls as MergeClairGVCFs {
+        input:
+            vcfs     = Clair.gvcf,
+            ref_dict = ref_dict,
+            prefix   = prefix + ".clair.g"
+    }
+
+    call VariantUtils.MergePerChrCalls as MergeClairVCFs {
+        input:
+            vcfs     = Clair.vcf,
+            ref_dict = ref_dict,
+            prefix   = prefix + ".clair"
+    }
+
     output {
         File dvp_phased_vcf = MergeDeepVariantPhasedVCFs.vcf
```

Comment:
I think we can delete this for now.
I do have another one that will get DVP back.